mixer/adapter/stackdriver/log/log.go

```diff
@@ -275,11 +278,22 @@ func toReq(mapping *config.Params_LogInfo_HttpRequestMapping, variables map[stri
 		req.Latency = latency
 	}
 
-	req.LocalIP = fmt.Sprintf("%v", variables[mapping.LocalIp])
-	req.RemoteIP = fmt.Sprintf("%v", variables[mapping.RemoteIp])
+	req.LocalIP = toStringFromIP(variables[mapping.LocalIp])
+	req.RemoteIP = toStringFromIP(variables[mapping.RemoteIp])
 	return req
 }
 
+func toStringFromIP(v interface{}) string {
```

Comment:
Can we use Stringfy fucntion here? https://github.com/istio/istio/blob/master/mixer/pkg/adapter/instanceUtil.go#L34

Reply:
Thanks for pointing.. It didn't work as is.. but I changed the Stringify function to take care of byte[] case.